modules/extensions-soap-support/src/main/java/org/mule/runtime/module/extension/soap/internal/runtime/connection/ForwardingSoapClientConnectionProvider.java

```diff
@@ -82,4 +86,18 @@ public void disconnect(ForwardingSoapClient connection) {
   public ConnectionValidationResult validate(ForwardingSoapClient connection) {
     return success();
   }
+
+  @Override
+  public void initialise() throws InitialisationException {
+    if (transportProvider instanceof Initialisable) {
+      ((Initialisable) transportProvider).initialise();
+    }
+  }
+
+  @Override
+  public void dispose() {
+    if (transportProvider instanceof Disposable) {
```

Comment:
not just this but all phases should be applied. LifecycleUtils is your friend for that